rollingpin/args.py

```diff
@@ -1,13 +1,24 @@
 import argparse
+import os
+import sys
 
 
 PAUSEAFTER_DEFAULT = 1
 
 
 class ExtendList(argparse.Action):
+    def __init__(self, *args, **kwargs):
+        super(ExtendList, self).__init__(*args, **kwargs)
+        if isinstance(self.default, list):
+            self.true_default = self.default[:]
+        else:
+            self.true_default = self.default
 
     def __call__(self, parser, namespace, values, option_string):
         list_to_extend = getattr(namespace, self.dest)
+        if self.true_default:  # if a default is defined, remove it before extending
+            for default in self.true_default:
+                list_to_extend.remove(default)
```

Comment:
I am not a very big fan of this hackery but it is necessary for defaults and lists to work.  Long story short, the `self.default` variable seems to be overwritten on each value extension (at least in the test suite) so I am storing the original default in an untouched variable then using that.  Before this, the default was getting set in the namespace and never going away (so you didn't override on the CLI, just extend).  The name `true_default` is a pretty passive aggressive name I used in frustration because of the moving target nature, so if you think I should use a nicer name, let me know...